functorch/_src/vmap.py

```diff
@@ -86,12 +83,17 @@ def _create_batched_inputs(
                 f'of dimensionality {arg.dim()} so expected in_dim to satisfy '
                 f'0 <= in_dim < {arg.dim()}.')
 
-    batch_size = _validate_and_get_batch_size(flat_in_dims, flat_args)
+    return (_validate_and_get_batch_size(flat_in_dims, flat_args), flat_in_dims, flat_args, args_spec)
```

Comment:
nit: outer parentheses aren't needed (`return a, b` in python returns a tuple `(a, b)`)